conf/application.conf

```diff
@@ -28,11 +28,14 @@ play.http.requestHandler = "uk.gov.hmrc.play.bootstrap.http.RequestHandler"
 # Provides an implementation of AuditConnector. Use `uk.gov.hmrc.play.bootstrap.AuditModule` or create your own.
 # An audit connector must be provided.
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuditModule"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
 
 # Provides an implementation of MetricsFilter. Use `uk.gov.hmrc.play.graphite.GraphiteMetricsModule` or create your own.
 # A metric filter must be provided
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModule"
 
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
```

Comment:
Duplicate of line 31?

Reply:
Yes, it is. That's weird. But undeniable. :)

Reply:
Fixed it now, not sure how that happened!